### PR TITLE
Fixes #194 : Add option to select all images in the image selector

### LINKED ISSUE #194: Add option to select all images in the image selector
**Actual Behaviour**

There is no option to select all images in any image selector.

**Expected Behaviour**

There should be option on the toolbar on long pressing an image to select all the images at once.

**Screenshots of the issue**

screenshot_2017-03-20-00-16-40-759_vn mbm phimp me

**Would you like to work on the issue?**

Yes, I would like to work on it.

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #194 

Screenshots for the change: 

screenshot_2017-03-28-00-01-26-720_vn mbm phimp me

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @rohanagarwal94 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/rohanagarwal94/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java [hunk 1]
```diff
@@ -3551,23 +3566,31 @@ else if(c<=20){
...
-			return images.size();
+			return localImageList.size();
 		}
 
 		public Object getItem(int position) {
 			return position;
 		}
 
+        public void selectAllImages()
+        {
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: Move the brace in the same line of the function

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java [hunk 2]
```diff
@@ -513,6 +515,19 @@ public void onClick(DialogInterface dialog, int id) {
                         });
                 // Create the AlertDialog object and show it
                 deleteAlert.create().show();
+            case R.id.menu_gallery_select_all:
+                photosAdapter.selectAllImages();
+                int imgCount = localImageList.size();
+                String newTitle;
+                if (imgCount > 0) {
+                    newTitle = imgCount + " images" + " selected";
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: It will be **images** in case of only one image.

**Comment by rohanagarwal94** [comment]
@pa1pal made the necessary changes. Please review.

**Pull request merged**